woocommerce/woocommerce-android
# "Create customer" endpoint for active plugin

Merge instructions:

1. Review this PR
2. Make sure https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2227 is merged
3. Update fluxc hash in this PR
4. Remove "Do not merge" label
5. Merge this PR

Closes: #5338 


### Description

This PR updates WCAndroid so it's compatible with fluxC after introduction of `wc/v3/wc_stripe/orders/<order_id>/create_customer` endpoint - https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2227

### Testing instructions

1. Install the newest version of the Stripe Extension plugin to your store (the version hasn't been released yet, you'll need to build it locally)
2. Make sure WCPay is not active in your store
3. Open the App
4. Navigate to the Order detail screen which is eligible for IPP
5. Tap on "Collect Payment" button
6. Check in the debugger that `wc/v3/wc_stripe/orders/<order_id>/create_customer` endpoint is invoked. You can use a flipper or a similar tool for this as well.
